arm-compute/2016-03-30/swagger/compute.json

```diff
@@ -4561,6 +4681,14 @@
         }
       },
       "x-ms-azure-resource": true
+    },
+    "GenericOkResponse": {
```

Comment:
Can we have a better name for the model. This will be the return type for all the POST (action) operations?
How about `OperationStatus` ?